### PR TITLE
Expand qualification abbreviations on first instance eg pgce and qts

### PR DESCRIPTION
**Pull request by avinhurry**
### Context

We are changing our course qualification abbreviations for the first instance. 

### Changes proposed in this pull request

- Expose `extended_qualification_descriptions` method to V3 Serializable Course
- Define `extended_qualification_descriptions` method to use in place of `description` method
- Fix tests

### Guidance to review

- Pull down this PR and the [respective PR](https://github.com/DFE-Digital/find-teacher-training/pull/1620
) for Find locally. 
- Navigate to courses with different qualification
- Ensure the expanded description is correct.

### Screenshots

image
image
image
image
image

_________________________________________________________________________________________________________

image

image
image
image
image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by avinhurry** [comment]
[Expand abbreviations on first instance (e.g., PGCE and QTS)](https://trello.com/c/yVRt6p0N/828-expand-abbreviations-on-first-instance-eg-pgce-and-qts)

File: app/models/concerns/with_qualifications.rb [hunk 1]
```diff
@@ -60,12 +60,28 @@ def qualifications
       end
     end
 
+    def full_qualifications
```
Comment (defong) [inline app/models/concerns/with_qualifications.rb:63]: Is there point to the array?

Comment (avinhurry) [reply app/models/concerns/with_qualifications.rb:63]: No point at all.

https://github.com/DFE-Digital/publish-teacher-training/pull/3148/commits/6efd1eb0475b0753f472aeaffb7e27be30622a72

File: app/models/concerns/with_qualifications.rb [hunk 2]
```diff
@@ -60,12 +60,28 @@ def qualifications
...
+    end
+
     def qualifications_description
       return "" unless qualifications
 
       qualifications.map(&:upcase).sort.join(" with ")
     end
 
+    def full_qualification_descriptions
+      return "" unless full_qualifications
```
Comment (defong) [inline app/models/concerns/with_qualifications.rb:80]:
```suggestion
      "" || full_qualifications
```

Maybe

Comment (avinhurry) [reply app/models/concerns/with_qualifications.rb:80]: Had a chat offline, keeping it as is because it's easier on the brain to read.

**Pull request merged**